content/pixel_fifo.md

```diff
@@ -0,0 +1,252 @@
+# Pixel FIFO
+Before we get started, all references to a cycle are meant as T-cycles
+(4.19 MHz) and cycle counts are doubled on CGB in double speed mode. Also
+when stating that a certain action "lengthens mode 3" this means that
+mode 0 (hblank) is shortened. The extra time has to be made up somewhere
+and the lengthening of mode 3 eats into the mode that comes afterwards,
+which happens to be mode 0 as shown in the following diagram.
+
+![](imgs/game-boy-lcd-refresh-diagram.png "imgs/game-boy-lcd-refresh-diagram.png")
+
+## FIFO info
```

Comment:
This feels out of place. Instead of this could the info above be made into a yellow area as seen on the "Accessing VRAM and OAM" section from Video Display.md?

Reply:
I think this is the intention? 
```
::: warning BOXTITLE
```
produces a yellow warning box